opentelemetry-sdk/src/opentelemetry/sdk/metrics/_internal/export/__init__.py

```diff
@@ -171,6 +171,39 @@ def force_flush(self, timeout_millis: float = 10_000) -> bool:
         return True
 
 
+class InMemoryMetricExporter(MetricExporter):
+    """Implementation of :class:`.MetricExporter` that stores metrics in memory.
+
+    This class can be used for testing purposes. It stores the exported metrics
+    in a dictionary in memory, indexed by an auto-incrementing counter that can
+    be retrieved using the `metrics` attribute.
```

Comment:
```suggestion
    be retrieved using the `_counter` attribute.
```

Reply:
If you did intend to highlight the `metrics` attribute, you could rephrase the text to something like:

> This class is designed for testing purposes. It maintains exported metrics in the `metrics` dictionary attribute.